torch/linalg/__init__.py

```diff
@@ -12,55 +12,63 @@
 cholesky = _add_docstr(_linalg.linalg_cholesky, r"""
 linalg.cholesky(input, *, out=None) -> Tensor
 
-Computes the Cholesky decomposition of a Hermitian (or symmetric for real-valued matrices)
-positive-definite matrix or the Cholesky decompositions for a batch of such matrices.
-Each decomposition has the form:
+Computes the Cholesky decomposition of a complex Hermitian (or real symmetric) positive-definite matrix or batch of matrices.
```

Comment:
"(or real symmetric)" -> "or real symmetric"

Reply:
Corrected throughout the docs